clang/test/SemaSYCL/double-precision-warn.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64 -internal-isystem %S/Inputs -fsyntax-only -sycl-std=2020 -verify %s -verify-ignore-unexpected=note
```

Comment:
```suggestion
// RUN: %clang_cc1 -fsycl-is-device -triple spir64 -internal-isystem %S/Inputs -fsyntax-only -sycl-std=2020 -verify %s
```

Is that '-verify-ignore-unexpected=note' needed?